website/docs/d/zones.html.md

```diff
@@ -83,6 +83,14 @@ resource "cloudflare_zone_lockdown" "endpoint_lockdown" {
     value = "198.51.100.4"
   }
 }
+
+resource "cloudflare_zone" "example" {
+  zone_id     = lookup(data.cloudflare_zones.test.zones[0], "id")
+  name        = "www"
+  value       = "192.168.0.11"
```

Comment:
Can we please use the reserved IP ranges for documentation from [rfc5737](https://tools.ietf.org/html/rfc5737)? 192.0.2.1 or 203.0.113.1 for example.

Reply:
Sure, updated.  I was just copying the examples from docs/resources/record